crates/ruff_linter/resources/test/fixtures/flake8_comprehensions/C419.py

```diff
@@ -13,6 +13,13 @@
 all(x.id for x in bar)
 any(x.id for x in bar)
 all((x.id for x in bar))
+# no lint if shadowed
+def all(x): pass
```

Comment:
Probably this deliberate shadowing should go below the existing `all()` and `any()` test cases at the bottom of this file?

Reply:
Or in a separate file to avoid mistakes adding cases in the future?

Reply:
Oh, I thought it was after all uses of `all`, but I missed the one inside `f()`. Which is a perfect illustration of why a separate file is better.